dali/base/dafdesc.cpp

```diff
@@ -82,7 +82,7 @@ bool getCrcFromPartProps(IPropertyTree &fileattr,IPropertyTree &props, unsigned
     }
     // NB: old @crc keys and compressed were not crc of file but of data within.
     const char *kind = props.queryProp("@kind");
-    if (kind&&strcmp(kind,"key")) // key part
+    if (kind&&streq(kind,"key")) // key part
```

Comment:
@jakesmith what effect would this have?

Reply:
it was the wrong way around.
The effect is that it would not be returning the crc values here when it should have been (should have been for non key types).
This is called from DFS in a few places, I think it going to mean if crc checking was on, it wouldn't work due to this.

Reply:
Might be worth changing to:
```
if (strsame(kind, "key"))
```